app/assets/stylesheets/fae/modules/forms/_radio.scss

```diff
@@ -7,7 +7,7 @@
 }
 
 .radio_collection--vertical {
-  label {
+  .radio label {
```

Comment:
This should be fixed! Good to know about the nesting and such. I'll keep my eyes out for further deviations 👍